src/server/routes.js

```diff
@@ -16,11 +16,12 @@ router.get('/', mainController.index);
 router.get('/payment-code', paymentCodeController.index);
 router.post('/payment-code', paymentCodeController.validatePaymentCode);
 router.get('/payment-code/:payment_code', paymentCodeController.getPaymentDetails);
+router.get('/payment-code/:payment_code/receipt', singlePaymentReceipt);
```

Comment:
Have you made the change yet to change the redirect URL we send to CPMS to this new route?

Reply:
I believe that redirect URL originates from public portal, so would expect to see it in this PR. Please point out if I've missed it!

Reply:
I don't think there's a link between cpms and receipts. There's a link from the penalty details page:

https://github.com/dvsa/rsp-public-portal/blob/849ad9a1b31ac81f16702177437b06db4afc316f/src/server/views/payment/paymentDetails.njk#L71

<img width="652" alt="screen shot 2018-12-13 at 09 40 55" src="https://user-images.githubusercontent.com/44976690/49929875-631d3c80-febb-11e8-9e66-0a5972265c34.png">

Reply:
As discussed, need to determine whether @md23 wants to return to single penalty receipt page after payment or keep the flow as it currently is (I'd imagine it's the former).